packages/api-client/src/components/CodeInput/CodeInput.test.ts

```diff
@@ -53,13 +54,25 @@ describe('CodeInput', () => {
     }
   })
 
+  const defaultProps = {
+    modelValue: 'test value',
+    environment: environmentSchema.parse({
+      uid: nanoid(),
```

Comment:
```suggestion
```
uids are automatically added!

Reply:
aaah not anymore as i get rid of the nanoid schema

Reply:
does it still after the last commit update?